packages/fxa-settings/src/models/integrations/base-integration.ts

```diff
@@ -133,6 +133,9 @@ export abstract class Integration<
       case MozServices.Pocket:
         return MozServices.Pocket;
 
+      case MozServices.TestService:
```

Comment:
Probably outside the scope of this PR, but I'd love to see this replaced with a datadriven approach.